airbyte-integrations/connectors/source-kyve/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 60a1efcc-c31c-4c63-b508-5b48b6a9f4a6
-  dockerImageTag: 0.2.2
+  dockerImageTag: 0.3.0
```

Comment:
@christopherbrumm you need to bump the project.toml version too and run poetry lock to update dependencies.